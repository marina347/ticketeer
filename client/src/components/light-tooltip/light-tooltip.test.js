import React from "react";
import { shallow } from "enzyme";
import LightTooltip from "./light-tooltip.component";

describe("LightTooltip component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<LightTooltip />);
  });

  it("Should render LighTooltip component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
