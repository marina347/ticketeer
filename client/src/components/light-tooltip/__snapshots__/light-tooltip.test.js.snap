// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LightTooltip component Should render LighTooltip component 1`] = `
<WithStyles(ForwardRef(Tooltip))
  classes={
    Object {
      "tooltip": "WithStyles(ForwardRef(Tooltip))-tooltip-1",
    }
  }
/>
`;
